lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -383,6 +383,13 @@ class TypeAnalysis : public dataflow::SparseDataFlowAnalysis<
   void visitOperation(Operation *op, ArrayRef<const ValueState *> operands,
                       ArrayRef<ValueState *> results) final;
 
+  void setToEntryState(ValueState *lattice) override {
+    if (lattice->isUninitialized())
+      return;
+    auto pessimisticState = ValueKnowledge::getPessimisticValueState(nullptr);
+    propagateIfChanged(lattice, lattice->join(pessimisticState));
```

Comment:
Just a guess, but should this just be `propagateIfChanged(lattice, lattice->reset())`?